aries-site/src/pages/templates/forms.mdx

```diff
@@ -161,6 +162,18 @@ All Form input elements need to be contained within a FormField.
   <ShippingExample />
 </Example>
 
+## Form Header
```

Comment:
Should this section be called something else? "Form Header" is just a placeholder in the design. The title should be aligned with what the form is demonstrating. In this case, we are demonstrating "Required and Optional Fields". Can re-word, but that is the essence.

See comment in other PR. I think this and the other examples being worked on deserve to live under a dedicated "Field and Form Validation" section.